module/VuFind/src/VuFind/RecordTab/CollectionList.php

```diff
@@ -63,6 +63,13 @@ class CollectionList extends AbstractBase
      */
     protected $recommendManager;
 
+    /**
+     * Backend id
+     *
+     * @var string
+     */
+    protected $backendId = 'SolrCollection';
```

Comment:
I would actually change this to `$searchClassId` -- SolrCollection is not a backend ID; it's a set of search objects that add extra abstraction on top of the Solr backend. I think using the backend terminology here might add confusion.

Reply:
Yes, renamed.